src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -2,33 +2,34 @@
 
 public class DefaultCountingOutRhymer {
 
-    private int[] numbers = new int[12];
+    private static final int MAX_STACK_SIZE = 12;
+    private static final int NO_ANY_ELEMENTS = -1;
+    private int[] numbers = new int[MAX_STACK_SIZE];
 
-    public int total = -1;
+    private int total = NO_ANY_ELEMENTS;
```

Comment:
Extracting this constant is done incorrectly. Should show during next classes ;)